packages/dbml-core/src/parse/Parser.js

```diff
@@ -54,6 +54,10 @@ class Parser {
     return new Parser().parse(str, format);
   }
 
+  static async fetchPostgresToJSON (connection) {
+    return fetch(connection, 'postgres');
```

Comment:
As mention above, please help test and make sure this work without problem on node 18, otherwise, we'll have to use a different name

Reply:
@huyphung1602 Remember to change this to `fetchSchema`